039-AKSForEnterprise/README.md

```diff
@@ -0,0 +1,43 @@
+# What The Hack - AKS Enterprise-Grade
+
+## Introduction
+
+This Hack has an objective studying in depth the integration surface between AKS and different Azure services. Some are optional components (for example monitoring), but others are required to successfully operate Kubernetes clusters on Azure (such as networking or storage).
+
+## Learning Objectives
+
+In this hack you will start by deploying a 3-tier application to AKS with very specific networking requirements.
+
+The complexity will quickly evolve towards security and storage, finishing with the last challenge focusing on Arc-enabled kubernetes clusters and arc-enabled data services.

```

Comment:
Please capitalize 'kubernetes' and 'arc'

Reply:
Done!